github/downloader_test.go

```diff
@@ -151,3 +164,107 @@ func TestOnlineOrganizationDownload(t *testing.T) {
 	}
 
 }
+
+func loadReqResp(filepath string, reqResp map[string]string) error {
+	// Open a RO file
+	decodeFile, err := os.Open(filepath)
+	if err != nil {
+		return err
+	}
+	defer decodeFile.Close()
+	// Create a decoder
+	decoder := gob.NewDecoder(decodeFile)
+	// Decode
+	decoder.Decode(&reqResp)
+	return nil
+}
+
+// RoundTripFunc .
+type RoundTripFunc func(req *http.Request) *http.Response
+
+// RoundTrip .
+func (f RoundTripFunc) RoundTrip(req *http.Request) (*http.Response, error) {
+	return f(req), nil
+}
+
+func TestOfflineOrganizationDownload(t *testing.T) {
+	var (
+		reqResp map[string]string = make(map[string]string)
+		err     error
+	)
+	// Load the recording
+	err = loadReqResp("../testdata/organization_src-d_2019-10-08.gob", reqResp)
```

Comment:
if you do
```
err := loadReqResp
```
then there is no need to define `err` above. But up to you.

Reply:
why not?
```go
require.NoError(t, loadReqResp("../testdata/organization_src-d_2019-10-08.gob", reqResp))
```